dash-pipeline/Makefile

```diff
@@ -237,6 +237,14 @@ sai-clean: SAI/SAI libsai-clean saithrift-server-clean
 	rm -rf SAI/SAI/inc SAI/SAI/experimental SAI/SAI/meta
 	cd SAI/SAI && git checkout -- inc experimental meta
 
+.PHONY:check-sai-spec
+check-sai-spec:
+	if git status | grep -q "SAI/specs"; then \
```

Comment:
What causes the exit status to change from 1 (fail) to 0 (pass)? If the developer commits headers locally for review does this somehow remove the failure?

Reply:
Hi Chris, thanks a lot for reviewing the change! 

This change is to only ensure all the SAI API changes caused by the PR will be reflected in PR as changes to SAI specs. So this task only runs in the CI pipeline and nowhere else. And the condition to pass the check is simple, as long as we have run 'make sai-headers' locally and adds the SAI spec updates in, this check will pass.

Reply:
ok, thanks!

Reply:
Thanks Chris!